app/src/main/kotlin/com/mapzen/erasermap/view/InitActivity.kt

```diff
@@ -0,0 +1,168 @@
+package com.mapzen.erasermap.view
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import android.animation.ObjectAnimator
+import android.app.AlertDialog
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.support.v7.app.AppCompatActivity
+import com.mapzen.erasermap.BuildConfig
+import com.mapzen.erasermap.CrashReportService
+import com.mapzen.erasermap.EraserMapApplication
+import com.mapzen.erasermap.R
+import com.mapzen.erasermap.model.RouterFactory
+import com.mapzen.erasermap.model.TileHttpHandler
+import com.mapzen.leyndo.ManifestDownLoader
+import com.mapzen.leyndo.ManifestModel
+import javax.inject.Inject
+
+public class InitActivity : AppCompatActivity() {
+
+    var apiKeys: ManifestModel? = null
+        @Inject set
+    var crashReportService: CrashReportService? = null
+        @Inject set
+    var routerFactory: RouterFactory? = null
+        @Inject set
+    var tileHttpHandler: TileHttpHandler? = null
+        @Inject set
+
+    var app: EraserMapApplication? = null
+    var tempKeys: ManifestModel? = null
+    var manifestRequestCount: Int = 0
+    var apiKeyFetchComplete: Boolean = false;
+
+    override public fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        app = application as EraserMapApplication
+        app?.component()?.inject(this)
+        initCrashReportService()
+        setContentView(R.layout.splash_screen)
+        getSupportActionBar().hide()
+        tempKeys = ManifestModel()
+        getApiKeys()
+        animate(0)
+    }
+
+    override public fun onStart() {
```

Comment:
No need to override lifecycle methods with no change in funcationality. We can delete `onStart()`, `onResume()`, `onPause()`, `onStop()`, and `onDestroy()` methods.